### PR TITLE
[Reviewer: AJH] Allow an external Chronos cluster to be configured

### PR DESCRIPTION
**Pull request by richardwhiuk**
Alex,

This is the only change required to sprout onto of the split-clusters work to allow Sprout to use an external storage node.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -422,6 +424,8 @@ static void usage(void)
        "                            The URI of this S-CSCF used by other servers, including AS, to contact\n"
        "                            this specific node. Defaults to \"sip:<localhost>:<port_scscf>\"."
        "     --pidfile=<filename>   Write pidfile\n"
+       "     --chronos-hostname <hostname>\n"
+       "                            Specify the hostname of a remote Chronos cluster\n"
```
Comment (AlexHockey) [inline src/main.cpp]: Say it defaults to localhost?

**Comment by AlexHockey** [comment]
Looks good. Just one comment inline.

**Comment by richardwhiuk** [comment]
@eleanor-merry - I think you've separately implemented this on dev and merged that into split clusters, so this can be closed?

**Comment by eleanor-merry** [comment]
Yup

**Pull request closed without merging**